src/components/user/uc/users.uc.js

```diff
@@ -61,20 +92,37 @@ const checkPermissions = async (id, roleName, { account }) => {
 	throw new Forbidden(`You don't have permissions to perform this action`);
 };
 
-const deleteUser = async (id, roleName, { account }) => {
+const deleteUser = async (id, roleName, { account, app }) => {
 	await checkPermissions(id, roleName, { account });
+	// const fileStorage = app.facade('/fileStorage/v2');
 
-	const userRelatedData = await getUserRelatedData(id);
+	const userAccountData = await getUserData(id);
+	const user = userAccountData.find(({ scope }) => scope === 'user').data;
 
-	const trashBin = await createUserTrashbin(id, userRelatedData);
+	await createUserTrashbin(id, userAccountData);
 
 	await replaceUserWithTombstone(id);
+	try {
+		const registrationPinFacade = app.facade('/registrationPin/v2');
```

Comment:
The facade can access without the app, please use this and do not pass the app into uc layer.